server-launch.sh

```diff
@@ -74,56 +73,57 @@ cd $scriptdir
 scriptdir=`pwd`
 
 data_dir=
+development=0
 ignore_error=0
 verbose=0
 clean_server=0
 location_tools=0
-port_access=127.0.0.1
 environment=
 launch_prefix=
-MAP_SERVER_PORT=9090
+: ${MAP_SERVER_PORT:=9090}
 profile=map
 
-while getopts "hd:E:p:fvcClP:" arg; do
+while getopts "hd:E:p:fvcCl" arg; do
```

Comment:
`-d` option does not have input value

```suggestion
while getopts "hdE:p:fvcCl" arg; do
```